spec/acceptance/server_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper_acceptance'
```

Comment:
I changed the name because the other PR will add more spec files for the other classes.